referenced/diseases.json

```diff
@@ -0,0 +1,1655 @@
+[
+    {
+        "id": 0,
+        "conceptType": "Disease",
+        "label": "Acute Lymphoid Leukemia",
+        "extensions": [
```

Comment:
I'd recommend storing `oncotree_term` and `oncotree_code` as `mappings`, similar to how you're representing genes

```json
{
                "coding": {
                    "id": "oncotree:ALL",
                    "code": "ALL",
                    "label": "Acute Lymphoid Leukemia",
                    "system": "https://oncotree.mskcc.org/?version=oncotree_latest_stable&field=CODE&search=",
                },
                "relation": "exactMatch",
            }
```